### PR TITLE
Cleared the unwanted files

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Deleted unwanted files in Algorithm course.

### CODE REVIEW

**Comment by udeysingh95** [comment]
I reordered some things on it. Please check them once.

File: GitHub/5-pull-and-push.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Push
+
+Using GitHub Desktop to PUSH to your local content to GitHub.Steps are:
+   * Click "Set up in a Desktop".
+   * Cloning in a GitHub Desktop.
+   * Copy all the required files from your local computer into the clone folder on your computer.
+   * Move to GitHub Desktop and commit to master.
+   * Publish branch in GitHub Desktop to upload your all files to GitHub.
+
```
Comment (udeysingh95) [inline GitHub/5-pull-and-push.md:28]: maybe a similar image here.

File: GitHub/5-pull-and-push.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+5. Publish the branch.
+6. Create a PULL Request.
+   6.1 You can now make a PULL request by clicking "Create pull request".
+   6.2 You can also now write a message and then click "Create pull request" again.
+7. In this final step, it’s time to bring your changes together – merging your readme-edits branch into the master branch.
+8. Click the green Merge pull request button to merge the changes into master.
+9. Click Confirm merge.
+10. Go ahead and delete the branch, since its changes have been incorporated, with the Delete branch button in the purple box.
+
+![Merge your pull request](https://guides.github.com/activities/hello-world/merge-button.png)
```
Comment (udeysingh95) [inline GitHub/5-pull-and-push.md:16]: I mistakenly moved these images to wrong chapter. Could you move them back to createPR file? both images

File: GitHub/5-pull-and-push.md (hunk 1 above)
Comment (Vaibhavpratapsingh22) [reply GitHub/5-pull-and-push.md:28]: Okay i will add

File: GitHub/5-pull-and-push.md (hunk 2 above)
Comment (Vaibhavpratapsingh22) [reply GitHub/5-pull-and-push.md:16]: No problem will be restored

**Pull request merged**